lib/chef/provider/package/chocolatey.rb

```diff
@@ -153,7 +153,7 @@ def get_choco_version
 
         # Choco V2 uses 'Search' for remote repositories and 'List' for local packages
         def query_command
-          return "list" if get_choco_version.match?(/^1/)
+          return "list" if get_choco_version.match?(/^1/) && !get_choco_version.match?(/^1.4/)
```

Comment:
This will break again when 1.5 comes out. We should be a bit more thorough here. Maybe something like...

```suggestion
          v_bits = get_choco_version.split('.')
          # 1.4+ and 2+
          if v_bits[0].to_i > 1 || v_bits[1] > 3
            return "search"
          end
          "list"
```

Reply:
@jaymzh I'll raise you a `Gem::Dependency.new('', '< 1.4.0').match?('', get_choco_version)`